internal/attestation/slsa_provenance_02_test.go

```diff
@@ -365,3 +366,78 @@ func encode(payload string) string {
 func buffy(data string) io.ReadCloser {
 	return io.NopCloser(bytes.NewBufferString(data))
 }
+
+func TestSLSAProvenance_Subject(t *testing.T) {
+	mockSubject1 := in_toto.Subject{
+		Name: "registry.io/example/image@sha256:abc123",
+		Digest: map[string]string{
+			"sha256": "abc123def456",
+		},
+	}
+	mockSubject2 := in_toto.Subject{
+		Name: "registry.io/example/artifact@sha256:def456",
+		Digest: map[string]string{
+			"sha256": "def456abc123",
+			"sha512": "xyz789uvw012",
```

Comment:
Nitpick: Even though it probably doesn't matter, it seems weird to have non-hex chars in the string here.